src/base/Notifier.sol

```diff
@@ -76,14 +75,12 @@ abstract contract Notifier is INotifier {
         delete subscriber[tokenId];
 
         if (address(_subscriber).code.length > 0) {
-            uint256 subscriberGasLimit = block.gaslimit.calculatePortion(BLOCK_LIMIT_BPS);
-
             // require that the remaining gas is sufficient to notify the subscriber
             // otherwise, users can select a gas limit where .notifyUnsubscribe hits OutOfGas yet the transaction/unsubscription
             // can still succeed
-            // to account for EIP-150, condition could be 64 * gasleft() / 63 <= subscriberGasLimit
-            if ((64 * gasleft() / 63) < subscriberGasLimit) GasLimitTooLow.selector.revertWith();
-            try _subscriber.notifyUnsubscribe{gas: subscriberGasLimit}(tokenId, config) {} catch {}
+            // to account for EIP-150, condition could be 64 * gasleft() / 63 <= unsubscribeGasLimit
+            if ((64 * gasleft() / 63) < unsubscribeGasLimit) GasLimitTooLow.selector.revertWith();
```

Comment:
I think we do not need to calculate the (64 * gasLeft / 63) check. 

Really this line is just being added to enforce some minimum amount that a user MUST send. 

That minimum amount doesn't exactly need to be 63/64ths of the unsubscribeGasLimit. It can just be at least the unsubscribeGasLimit. And we can add a comment that in 1 case (when gasLeft == gasLimit) the call will only forward 63/64ths of the gasLimit... which is fine. Basically we can just keep this in mind when parameterizing the `unsubscribeGasLimit`.

We really want to add this line because we do not want the user to ever send a minimal amount of gas such that the unsubscribe block reverts but the rest of the function succeeds